### ISSUE TITLE
Windows install fails because version unavailable on Bintray

### ISSUE THREAD

**Issue by emachnic** [chef-cookbooks/habitat#197]
### Cookbook version
1.5.0

### Chef-client version
15.0.300

### Platform Details
Windows 2012R2

### Scenario:
Trying to install the latest version of Habitat

### Steps to Reproduce:
Use the `hab_install` resource with the latest version of the cookbook.

### Expected Result:
Version 1.5.0 of Habitat should be installed

### Actual Result:
Install will fail with a `404 "Not Found"` because it can't find version 1.5.0

```
* hab_install[install Habitat] action install
           * remote_file[C:\Users\ADMINI~1\AppData\Local\Temp\kitchen\cache/hab--x86_64-windows.zip] action create[2020-01-30T13:00:25-06:00] WARN: remote_file[C:\Users\ADMINI~1\AppData\Local\Temp\kitchen\cache/hab--x86_64-windows.zip] cannot be downloaded from https://api.bintray.com/content/habitat/stable/windows/x86_64/hab--x86_64-windows.zip?bt_package=hab-x86_64-windows: 404 "Not Found"
```

**Comment by christophermaier** [comment]
We stopped using Bintray for packages a while back; this will need to be downloaded from packages.chef.io now, similar to other Chef binaries.

**Comment by christophermaier** [comment]
Alternatively, the [install.ps1](https://github.com/habitat-sh/habitat/blob/master/components/hab/install.ps1) script will work as well.

**Comment by jonlives** [comment]
Fixed this in #198 and released v1.5.1 of the cookbook this morning.
